### PR TITLE
[Data Factory]Add schema mapping for hierarchical data

### PR DESCRIPTION
**Pull request by charlesshen-msft**

### CODE REVIEW

File: specification/datafactory/resource-manager/Microsoft.DataFactory/preview/2017-09-01-preview/entityTypes/Pipeline.json [hunk 1]
```diff
@@ -1616,6 +1616,10 @@
         "columnMappings": {
           "type": "object",
           "description": "Column mappings. Type: string (or Expression with resultType string)."
+        },
```
Comment (lmazuel) [inline specification/datafactory/resource-manager/Microsoft.DataFactory/preview/2017-09-01-preview/entityTypes/Pipeline.json]: Could you update or add an example for that?

Comment (lmazuel) [reply specification/datafactory/resource-manager/Microsoft.DataFactory/preview/2017-09-01-preview/entityTypes/Pipeline.json]: I meant, x-ms-example, like this:
https://github.com/iamapi/azure-rest-api-specs/tree/a3c2bff7ee233f1de44032b1fccc27450874c77f/specification/datafactory/resource-manager/Microsoft.DataFactory/preview/2017-09-01-preview/examples

File: specification/datafactory/resource-manager/Microsoft.DataFactory/preview/2017-09-01-preview/entityTypes/Pipeline.json [hunk 2]
```diff
@@ -1615,7 +1615,11 @@
       "properties": {
         "columnMappings": {
           "type": "object",
-          "description": "Column mappings. Type: string (or Expression with resultType string)."
+          "description": "Column mappings. Example: \"UserId: MyUserId, Group: MyGroup, Name: MyName\" Type: string (or Expression with resultType string)."
```
Comment (lmazuel) [inline specification/datafactory/resource-manager/Microsoft.DataFactory/preview/2017-09-01-preview/entityTypes/Pipeline.json]: I meant, x-ms-example, like this:
https://github.com/iamapi/azure-rest-api-specs/tree/a3c2bff7ee233f1de44032b1fccc27450874c77f/specification/datafactory/resource-manager/Microsoft.DataFactory/preview/2017-09-01-preview/examples

**Review by lmazuel — CHANGES_REQUESTED** [review]

**Comment by lmazuel** [comment]
I'm about to close this PR for inactivity. Friendly reminder. Thank you.

**Comment by hvermis** [comment]
@lmazuel Please wait for couple days. I think our China team is OOF

**Review by lmazuel — APPROVED** [review]

**Pull request merged**